fpdf/output.py

```diff
@@ -555,6 +555,18 @@ def _add_fonts(self):
                     cmap[unicode] for unicode in uni_to_new_code_char if unicode in cmap
                 ]
 
+                missing_glyphs = [
+                    chr(unicode)
+                    for unicode in uni_to_new_code_char
+                    if unicode not in cmap
+                ]
+                if len(missing_glyphs) > 0:
+                    LOGGER.warn(
```

Comment:
```suggestion
                    LOGGER.warning(
```
_cf._ https://docs.python.org/3/library/logging.html#logging.Logger.warning